index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="main.css">
+    <title>Portfolio-Mobile</title>
+</head>
+
+<body>
+    <header>
+        <h1>Jonh Doe</h1>
+        <nav>
+            <img src="pictures/Enabled.png" alt="menu">
+        </nav>
+    </header>
+    <section class="main">
```

Comment:
- [ ] This section with class `main` has not been targeted in the CSS. It should be styled to align its content with Flexbox as per the requirements. Please work on it.